cmds/core/mv/mv.go

```diff
@@ -59,25 +61,31 @@ func moveFile(source string, dest string) error {
 			return nil
 		}
 	}
-
 	if err := os.Rename(source, dest); err != nil {
-		return err
+		// Check for EXDEV; if so, we'll do a CopyTree
+		uerr, ok := err.(*os.LinkError)
+		if !ok || uerr.Unwrap() != unix.EXDEV {
```

Comment:
I would use errors.Is(err, unix.EXDEV), if possible? No need to cast, I *think* it'll go unwrap all that stuff by itself.

```go
if !errors.Is(err, unix.EXDEV) {
  return err
} 
if err := cp....
```

Reply:
I'll give that a try, thanks!